src/python/tests/utils.py

```diff
@@ -0,0 +1,177 @@
+#
+# This file is part of udmp-parser project
+#
+# Released under MIT License, by 0vercl0k - 2023
+#
+# With contribution from:
+# * hugsy - (github.com/hugsy)
+#
+
+import pathlib
```

Comment:
Hum maybe a quick `black` on this file? The imports are not sorted, very disturbing 😄

Reply:
weird I thought I ran it, let me see

Reply:
```
  File "c:\Users\over\Downloads\udmp-parser\src\python\tests\utils.py", line 166, in <module>
    ) -> Optional[tuple[int, pathlib.Path]]:
TypeError: 'type' object is not subscriptable
```

Hmmm lemme see..

Reply:
ohhhh damn, maybe we need to bump the python version

Reply:
Hmm or do we need to `from typing import Tuple` / `Tuple[]`?